package.json

```diff
@@ -67,6 +67,7 @@
     "chalk": "^2.2.0",
     "cross-env": "^5.0.5",
     "css-loader": "^6.7.1",
+    "dugite": "2.2.0",
```

Comment:
This was accidentally added to package.json instead of app/package.json. Gonna take care of that and request a review from @sergiou87